src/components/avatar/avatar.jsx

```diff
@@ -0,0 +1,38 @@
+import { ThemeProvider } from "styled-components";
+import styled from "styled-components";
+
+/**
+ * Avatar Component
+ * @author [J. Hartsek](https://github.com/JHartsek)
+ * @param shape expects 'round' or 'square'
+ * @param src the image link
+ * @param size expects 'xs', 'small', 'medium', or 'large'
+ * @param alt the image description
+ */
+
+const Container = styled.img`
+  height: ${(props) => props.theme[props.size]};
+  width: ${(props) => props.theme[props.size]};
+`;
+
+export const Avatar = ({ shape, src, size, alt }) => {
+  const sizes = {
+    xs: "35px",
+    small: "40px",
+    medium: "50px",
+    large: "60px",
+  };
+
+  return (
+    <ThemeProvider theme={sizes}>
```

Comment:
This certainly isn't wrong-- but it may be unnecessary to use a ThemeProvider here; The code may be a little simpler to read if you simply pass a size into `Container` like this:
```
size={sizes[size]}
```

and then inside of `Container` you can do `${(props) => props.size};`